### ISSUE TITLE
Allow quotation marks replacement disabling

### ISSUE THREAD

**Issue by DenMesh** [randy3k/LaTeXYZ#8]
First of all, a word of thanks. Your package is truly a lifesaver in math mode. However, the decision to autoreplace quotation marks was... questionable, IMO. For example, it plays havok with TiKZ and csquotes. It would have been tolerable if `auto_match_enabled` switched this off, but this does not happen. It does switch off replacement without selection, but when wrapping selection in quotes it still replaces them.

TL; DR
1.  `auto_match_enabled=false` still replaces quotes when wrapping selection
2. Would really like a separate switch for quote replacement

**Comment by randy3k** [comment]
You could always redefine the key binds if you don't like them. Just add the default in your user settings file.

**Comment by randy3k** [comment]
0.1.7 has a setting `use_latex_quotes` to disable this.
